internal/pkg/panic-recovery/panic_recovery.go

```diff
@@ -23,20 +23,21 @@ func CollectPanic(cmd *cobra.Command, args []string, cfg *v1.Config) *usage.Usag
 		Arch:        cliv1.PtrString(runtime.GOARCH),
 		Version:     cliv1.PtrString(cfg.Version.Version),
 		Command:     cliv1.PtrString(fullCommand.CommandPath()),
-		Flags:       parseFlags(fullCommand, flags),
+		Flags:       ParseFlags(fullCommand, flags),
 		Error:       cliv1.PtrBool(true),
 		StackFrames: parseStack(string(debug.Stack())),
 	}
 }
 
-// parseFlags collects the flags alongside the panicking command
-func parseFlags(cmd *cobra.Command, flags []string) *[]string {
+// ParseFlags collects the flags of a command after being found with Find()
+func ParseFlags(cmd *cobra.Command, flags []string) *[]string {
```

Comment:
Any reason we're returning a pointer to a slice? In memory, a slice is already represented by a pointer to the first element so this is doing nothing.

Reply:
In the function above, `CollectPanic`, the usage struct takes in `*[]string` for `Flags` and `StackFrames`. Returning a pointer to a slice here and below in `parseStack` means I don't have to declare a variable in `CollectPanic` for either function call.

Reply:
For whatever reason I can't assign the field in the Usage constructor like `... Flags: &ParseFlags(),` which I think might just be a Go syntax thing

Reply:
> I don't have to declare a variable in CollectPanic for either function call

IMO, doing this definitely sounds better than returning a pointer to a slice

Reply:
Okay, sounds good!